apps/graphql/src/types/output/Segment.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { GraphQLObjectType, GraphQLInt } from 'graphql';
+import GlobalID from '@kiwicom/graphql-global-id';
+
+import DateType from './DateType';
+import Vehicle from './Vehicle';
+import Transporter from './Transporter';
+import Location from './Location';
+
+export default new GraphQLObjectType({
+  name: 'Segment',
+  fields: {
+    arrivalTime: { type: DateType },
+    departureTime: { type: DateType },
+    destination: { type: Location },
+    duration: { type: GraphQLInt },
+    id: GlobalID(({ id }) => id),
```

Comment:
can you move ID to the top, please?

Reply:
All other can be sort alphabetically is nice but the ID should be always as first.